site/src/sections/Navbar/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { NavStyleWrapper } from "./navbar.style";
+import pattern from "../../assets/images/brand/SVG/service-mesh-patterns-side.svg";
+import Layer5 from "../../assets/images/brand/SVG/Layer5.svg";
+const Navbar = () => {
+  return (
+    <NavStyleWrapper>
+      <nav>
+        <ul className="navbar">
+          <img className="pattern" src={pattern} alt="pattern" />
+          <li className="text">Book Chapters</li>
+          <li className="text">Patterns In-Action</li>
+          <li className="text">Meet the Authors</li>
+          <li className="text">About <img className="pattern" src={Layer5} alt="Layer5" /> </li>
```

Comment:
We don't really need a className here. All are `li` can be used directly in the CSSS as `li`

Reply:
:+1: